public/style/main.css

```diff
@@ -12,8 +12,24 @@ body {
 
 /*bs modification*/
 .btn {
-  border-radius    : 0px;
+  /*border-radius    : 0px;*/
+  border-radius    : 50px;
+  font-size: 1em;
```

Comment:
em only if we are going to be strict with it:

em Relative to the font-size of the element (2em means 2 times the size of the current font)

Thus, em needs to be consistently managed throughout. That means keeping track of em at all tiers of html